Project/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs

```diff
@@ -19,23 +18,13 @@ public override void InitializeAgent()
         base.InitializeAgent();
         m_AgentRb = GetComponent<Rigidbody>();
         m_MyArea = area.GetComponent<PyramidArea>();
-        m_RayPer = GetComponent<RayPerception>();
         m_SwitchLogic = areaSwitch.GetComponent<PyramidSwitch>();
     }
 
     public override void CollectObservations()
     {
         if (useVectorObs)
         {
-            const float rayDistance = 35f;
-            float[] rayAngles = { 20f, 90f, 160f, 45f, 135f, 70f, 110f };
```

Comment:
I matched the maxRayDegrees in the new component to the min/max of the old ones, but the exact distribution won't be the same.

Reply:
does it need to be retrained?

Reply:
Yes, new Pyramids.nn file is in PR. I ran it last night.